Cargo.toml

```diff
@@ -3,5 +3,5 @@ resolver = "2"
 members = [
   "birb",
   "birb_window",
-  "birb_winit",
+  "birb_winit", "registry",
```

Comment:
could the crate be renamed to `birb_registry` otherwise it will conflict on crates.io when we come to publish